src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -47,6 +47,7 @@
                 "JobPollingFrequency": "00:00:10",
                 "MaximumNumberOfResourcesPerQuery": 100,
                 "NumberOfPagesPerCommit": 10,
+                "MaxFileSizeInMB": 64,
                 "Formats": [
                     {
                         "Name": "resourcename_only",
```

Comment:
Should we add \<sequence\> to these Formats as well?

Reply:
Like discussed below, have updated code to add sequence irrespective of format.